src/home_robot_hw/home_robot_hw/env/visualizer.py

```diff
@@ -278,3 +278,211 @@ def _init_vis_image(self, goal_name: str):
         # vis_image[537 : 537 + lx, 155 : 155 + ly, :] = self.legend
 
         return vis_image
+
+
+class ExplorationVisualizer:
```

Comment:
I would really like to combine this with the other one somehow so we dont have so much duplicated code...

@theophilegervet  @yvsriram  we talked about deleting these and combining with the habitat one right? any thoughts?